src/qibosoq/programs/flux.py

```diff
@@ -84,3 +86,16 @@ def body(self):
         self.set_bias("zero")
         self.soc.reset_gens()
         self.sync_all(self.relax_delay)
+
+    def declare_zones_and_ro(self, sequence: List[Pulse]):
+        """Declare all nqz zones and readout frequencies.
```

Comment:
Just for myself: what it an _nqz zone_?

And, btw, why it's declaring all qubit channels but feedback?

Reply:
Nyquist zone (defined as intervals of fs/2, where fs is the sampling frequency). So basically from 0 to fs/2 is the first and fs/2 to fs is the second and so on.
This declaration is needed because for the first zone the synthesis works differently than for the other zones.

I am declaring it for all the generator channels, the adc does not need it (but needs the declaration of frequency for the downsampling)